examples/mock/mockoon-bb-information-mediator.json

```
@@ -79,6 +80,20 @@
               "value": "^[a-zA-Z\\d]+$",
               "invert": false,
               "operator": "regex"
+            },
+            {
+              "target": "query",
+              "modifier": "serviceId",
+              "value": "^([a-zA-Z][a-zA-Z\\\\d_:\\\\-.]*)$",
```

```suggestion
              "value": "^([a-zA-Z][a-zA-Z\\d_:\\-.]*)$",
```

Please change regex in mockoon to ```^([a-zA-Z][a-zA-Z\d_:\-.]*)$```